__tests__/app.test.js

```diff
@@ -51,3 +51,38 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe.only("GET /api/users", () => {
+  test("returns an array from a get request at this endpoint", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toBeInstanceOf(Array);
+      });
+  });
+  test("Elements of array are objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((res) => {
+        res.body.forEach((element) => {
+          expect(element).toBeInstanceOf(Object);
+        });
+      });
+  });
```

Comment:
These tests probably arent needed as you can cover them in the single test below

Reply:
deleted unneeded tests